Libraries/JavaScriptAppEngine/Initialization/InitializeJavaScriptAppEngine.js

```diff
@@ -70,20 +70,24 @@ function setUpConsole(): void {
  */
 function defineProperty(object: Object, name: string, newValue: mixed): void {
   const descriptor = Object.getOwnPropertyDescriptor(object, name);
-  if (descriptor) {
+  const {enumerable, writable, configurable} = descriptor || {};
+
+  if (descriptor && configurable) {
```

Comment:
Even if the original property was not configurable we should be able to set the backup property.